Big-Spender/readme.md

```diff
@@ -43,17 +43,22 @@ You are working with Claire and Farnoosh, who are trying to complete a missing r
 
 **You:** I can try. What kind of data are you looking for exactly?
 
+big-spender=# select * from spends where date between '2021-01-01' and '2021-12-31';    
+
 **Farnoosh:** We need to find out the transactions between £30,000 and £31,000. Could you help us write a query for that?
 
 **You:** Absolutely. Here's the SQL query you need:
+big-spender=# select * from spends where date between '2021-01-01' and '2021-12-31' and amount  between 30000 and 31000;    
```

Comment:
I like how you used the date and also the amount! This is good practice to be more specific. What else could you use instead of between? (not a problem, just thinking of other ways!)

Reply:
thank you @kwebster77 for reviewing my code, I think it can be 
select * from spends where date between '2021-01-01' and '2021-12-31' and amount  >=30000 and amount <= 31000;